File: Linguagem C Exercicios/C-Sequencial/TROCO.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

    double precoUn,precoTotal,recebido,troco = 0;
    int qnt = 0;

    printf("Preço unitario do produto: R$ ");
    scanf("%lf",&precoUn);
    printf("Quantidade comprada: ");
    scanf("%d",&qnt);
    printf("Dinheiro recebido: ");
    scanf("%lf",&recebido);

    precoTotal = qnt * precoUn;
    troco = recebido - precoTotal;

    printf("TROCO = %.2lf",troco);

    return 0;
}

File: Linguagem C#/Sequencial/Troco.cs
using System;
using System.Globalization;

namespace Troco {
    class Program {
        static void Main(string[] args) {

            CultureInfo CI = CultureInfo.InvariantCulture;

            double precoUn, dinheiroRecebido, troco, precoTotal;
            int qnt;

            Console.Write("preco unitario do produto: ");
            precoUn = double.Parse(Console.ReadLine(),CI);
            Console.Write("Quantidade comprada: ");
            qnt = int.Parse(Console.ReadLine());
            Console.Write("Dinheiro recebido: ");
            dinheiroRecebido = double.Parse(Console.ReadLine(),CI);

            precoTotal = precoUn * qnt;
            troco = dinheiroRecebido - precoTotal;

            Console.WriteLine("TROCO = "+troco.ToString("F2",CI));
        }
    }
}
